composer.json

```diff
@@ -110,82 +110,94 @@
+        "tecnickcom/tcpdf": "^6.8.2",
+        "twig/extra-bundle": "^3.20",
+        "twig/string-extra": "^3.20",
+        "twig/twig": "^3.20.0",
         "wallabag/phpepub": "^4.0.10",
         "wallabag/rulerz": "dev-master",
         "wallabag/rulerz-bundle": "dev-master",
-        "willdurand/hateoas": "^3.10",
-        "willdurand/hateoas-bundle": "^2.6"
+        "willdurand/hateoas": "^3.12",
+        "willdurand/hateoas-bundle": "^2.7"
     },
     "require-dev": {
-        "dama/doctrine-test-bundle": "^8.0.2",
-        "doctrine/data-fixtures": "^1.7",
-        "doctrine/doctrine-fixtures-bundle": "^3.5.1",
-        "ergebnis/composer-normalize": "^2.42.0",
-        "friendsofphp/php-cs-fixer": "^3.49",
-        "friendsoftwig/twigcs": "^6.1",
+        "dama/doctrine-test-bundle": "^8.2.2",
+        "doctrine/data-fixtures": "^1.8.1",
+        "doctrine/doctrine-fixtures-bundle": "^3.7.1",
+        "ergebnis/composer-normalize": "^2.45.0",
+        "friendsofphp/php-cs-fixer": "^3.70.2",
+        "friendsoftwig/twigcs": "^6.5",
         "m6web/redis-mock": "^5.6",
-        "php-http/mock-client": "^1.6",
-        "phpstan/extension-installer": "^1.3.1",
-        "phpstan/phpstan": "^1.10.59",
-        "phpstan/phpstan-doctrine": "^1.3.62",
-        "phpstan/phpstan-phpunit": "^1.3.16",
-        "phpstan/phpstan-symfony": "^1.3.7",
-        "phpunit/phpunit": "^9.6.17",
-        "shipmonk/composer-dependency-analyser": "^1.7",
-        "symfony/css-selector": "^5.4.35",
-        "symfony/debug-bundle": "^5.4.35",
-        "symfony/maker-bundle": "^1.43",
-        "symfony/phpunit-bridge": "^7.0.3",
-        "symfony/process": "^5.4",
-        "symfony/var-dumper": "^5.4.35",
-        "symfony/web-profiler-bundle": "^5.4.35",
+        "php-http/mock-client": "^1.6.1",
+        "phpstan/extension-installer": "^1.4.3",
+        "phpstan/phpstan": "^1.12.20",
+        "phpstan/phpstan-doctrine": "^1.5.7",
+        "phpstan/phpstan-phpunit": "^1.4.2",
+        "phpstan/phpstan-symfony": "^1.4.13",
+        "phpunit/phpunit": "^9.6.22",
+        "shipmonk/composer-dependency-analyser": "^1.8.2",
+        "symfony/css-selector": "^5.4.45",
+        "symfony/debug-bundle": "^5.4.45",
+        "symfony/maker-bundle": "^1.50",
+        "symfony/phpunit-bridge": "^7.2.0",
+        "symfony/process": "^5.4.47",
+        "symfony/var-dumper": "^5.4.48",
+        "symfony/web-profiler-bundle": "^5.4.48",
         "symfony/web-server-bundle": "^4.4.44"
     },
+    "replace": {
+        "symfony/polyfill-php54": "*",
+        "symfony/polyfill-php55": "*",
+        "symfony/polyfill-php56": "*",
+        "symfony/polyfill-php70": "*",
+        "symfony/polyfill-php71": "*",
+        "symfony/polyfill-php72": "*",
+        "symfony/polyfill-php73": "*",
+        "symfony/polyfill-php74": "*",
+        "symfony/polyfill-php80": "*",
+        "symfony/polyfill-php81": "*",
+        "symfony/polyfill-php82": "*"
+    },
```

Comment:
What does it actually do?

Reply:
It avoid installing those packages in vendor. It basically says "hey Composer I'm already providing those packages myself with my code"